tests/report_test.sh

```diff
@@ -556,8 +560,12 @@ function test_save_data_to()
   assert_equals_helper 'We expect a directory path to be valid' "$LINENO" 0 "$ret"
 
   # Verifying that the correct message is displayed.
-  expected="The report output was saved in: ${SHUNIT_TMPDIR}/report_output"
-  assert_equals_helper 'We expect a valid message' "$LINENO" "$expected" "$output"
+  declare -a expected_cmd=(
+    "touch ${SHUNIT_TMPDIR} 2> /dev/null"
+    "The report output was saved in: ${SHUNIT_TMPDIR}/report_output"
+  )
+
+  compare_command_sequence 'We expect a valid message' "$LINENO" 'expected_cmd' "$output"
```

Comment:
As I commented above, you shouldn't have to make this adaptation as we are not testing a call to `save_data_to` passing the verbose flag.

That being said, you should add a dedicated case (maybe another function named `test_save_data_to_with_verbose`) to test the behaviour you introduced in this commit.